azavea/echo-locator
# Feature/kak/fit route details#106

## Overview

Zoom map to fit bounds, by default to the neighborhood polygon extent, or to the route to the currently selected neighborhood, if currently showing neighborhood details.


### Notes

[VectorGrid layers do not support getBounds](https://libraries.io/github/haoliangyu/Leaflet.VectorGrid#todo), which is why this adds a small library, `geojson-bbox`, to support finding the extent containing all the neighborhood polygons.


## Testing Instructions

 * `cd taui && yarn install && yarn start`
 * Navigate between detail and list views in sidebar
 * Navigate between detail views by clicking on map markers
 * Map should zoom to fit route for current detail view, if any
 * Map should zoom to fit all neighborhood boundaries on initial load


Closes #106
